src/System.Security.Cryptography.Cng/src/project.json

```diff
@@ -25,6 +26,11 @@
       "dependencies": {
         "Microsoft.TargetingPack.NETFramework.v4.6.1": "1.0.1"
       }
+    },
+    "net463": {
+      "dependencies": {
+        "Microsoft.TargetingPack.NETFramework.v4.6.2": "1.0.1"
```

Comment:
Is this a temporary thing, since we don't have a 4.6.3 targeting pack?  If so, please file an issue so we come back and fix this later.